Sources/Components/SaveSearchView/LabelSwitchView.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+protocol LabelSwitchViewDelegate: AnyObject {
+    func labelSwitchViewValueChanged(_ labelSwitchView: LabelSwitchView)
+}
+
+class LabelSwitchView: UIView {
```

Comment:
Is it possible to use `SwitchView` instead of making a new one?

Reply:
I will try it out, i didn't see it around

Reply:
So, it wasn't used anywhere relevant (both in FinniversKit and the finn app) just in `BottomSheetMechanicsDemoViewController` within FinniversKit

Then i suggest to replace the current SwitchView with this one

Reply:
Sounds good to me 👍